sandbox_windows.go

```diff
@@ -0,0 +1,12 @@
+// +build windows
```

Comment:
don't think this build tag is needed, because the file is already named `_windows.go`